API/Chapters.md

```diff
@@ -53,10 +53,8 @@ The endpoint URL may be defined by the chapter provider. The Podcast Hosting ser
 Parameters:
 - `rss`: RSS feed URL
 - `guid`: New Episode GUID
-- `enclosure_url`: New episode enclosure url
 
 Response:
-- `status`: true or false
```

Comment:
I think this can be handled by the HTTP response code, or does this indicate something else I'm missing?

Reply:
This was copied from the podcastIndex Apis but I agree with you, we should remove it.